frontend/src/components/Common/MultiAutoComplete/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Checkbox, createFilterOptions, TextField } from '@mui/material'
+import React from 'react'
+import { StyledAutocompleted } from './styles'
+
+type Props = {
+  optionList: string[]
+  selectedOption: string[]
+  // There is an any because m-ui strictly define its type, but the parameters are not that strict. Maybe because of version diff
+  onChangeHandler: any
```

Comment:
not recommend to use `any`, just a function, you can search online

Reply:
> not recommend to use `any`, just a function, you can search online

While...I do searched it but if I use the function with parameters type that the mui offical website provided, the type system would complain about it. That's why I left that comments above.

Reply:
> not recommend to use `any`, just a function, you can search online

I suspect it maybe  the dependency version issue?